popup/p.css

```diff
@@ -27,3 +27,7 @@ body:not(.loggedin) #new {
 .tag[data-signal="false"] .sigAgainstContainer {
     font-weight: bold;
 }
+
+.tag[data-signal="false"] .sigEraseContainer {
```

Comment:
Shouldn't `data-signal` be something else here? Something like `null` or `undefined`.

Reply:
Definitely, thanks for catching that :)